.evergreen/start-orchestration.sh

```diff
@@ -67,17 +67,21 @@ killport() {
 
   if [[ "${OSTYPE:?}" == cygwin || "${OSTYPE:?}" == msys ]]; then
     for pid in $(netstat -ano | grep ":$port .* LISTENING" | awk '{print $5}' | tr -d '[:space:]'); do
-      taskkill /F /T /PID "$pid" || true
+      kill "$pid" || true
+      timeout 60s bash -c "while kill -0 \"$pid\" 2>/dev/null; do sleep 1; done" || kill -9 "$pid" || true
     done
   elif [ -x "$(command -v lsof)" ]; then
     for pid in $(lsof -t "-i:$port" || true); do
       kill "$pid" || true
+      timeout 60s bash -c "while kill -0 \"$pid\" 2>/dev/null; do sleep 1; done" || kill -9 "$pid" || true
```

Comment:
The reason I changed it to SIGTERM was to give MO a chance to cleanup its mongo processes but that doesn't seem to work. If we're going back to `kill -9` (SIGKILL) instead of SIGTERM then can we remove the timeout logic?

Reply:
The use of `-9` in the while loop was a typo. `-0` was meant to be used instead. Please re-review.

Reply:
Quoting @nbbeeken from Slack discussions:

> The switch to sigterm was intentional because mongo-orch will try to clean up the servers gracefully, but maybe there's a bug in that logic and we should send -9 anyway :blob-sad:  you try to be a 'good citizen' and look what it gets you

If we want to revert to the simpler "just SIGKILL" routines we had before, that would also address the flaky "address already in use" problem this PR is trying to solve.

Reply:
In Jupyter we use the pattern of SIGTERM, wait on pid, SIGKILL as well.  I think it is a good pattern in general.

Reply:
Yeah I'd prefer SIGKILL at this point because it should save time (don't need to wait 60 seconds) and will make this code easier to read.